browser/pFa_test.go

```diff
@@ -0,0 +1,233 @@
+package browser
+
+import (
+	"github.com/vertgenlab/gonomics/dna/pDna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta/pFasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"testing"
+)
+
+var PrintOneSetLinesTests = []struct {
+	LineLength    int
+	SetOfLinesIdx int
+	NumIters      int
+	LineA         []float32
+	LineC         []float32
+	LineG         []float32
+	LineT         []float32
+	Start         int
+	Records       []pFasta.PFasta
+	Out           string
+	SigFigs       int
+	DecimalPlaces int
+	LongestName   int
+	Expected      string
+}{{LineLength: 10,
+	SetOfLinesIdx: 0,
+	NumIters:      6,
+	LineA:         make([]float32, 30),
+	LineC:         make([]float32, 30),
+	LineG:         make([]float32, 30),
+	LineT:         make([]float32, 30),
+	Start:         2,
+	Records: []pFasta.PFasta{pFasta.PFasta{Name: "chr1",
```

Comment:
pFaTools sample is essentially a pfasta-to-fasta. Craig had mentioned making a 'consensus' tool, which takes the consensus sequence instead of a posterior sample, might be another valuable future addition.

As for fasta-to-pfasta, I agree it'd be cool to make a tool that makes one-hot encoded pFasta sequences from a fasta sequence. No uncertainty, but this at least gets us to the right data type